### ISSUE TITLE
Move 0 steps doesn't show a dot in flat ends mode

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap#2770]
from forum

**Comment by jmoenig** [comment]
Yeah, but... of course! That‘s absolutely right. What should we do? Draw a square? Centered at the rotation center? But when the step is 0.0001 we don‘t center, right? And then people still wouldn‘t see any dot and complain. Naw, Snap! Is doing the perfectly right thing here. No special cases!

**Comment by brianharvey** [comment]
In a situation like this, our guide should be what the user wants.  I had to talk you into drawing a dot for MOVE 0 STEPS, and then you put it in, and neither of us thought of checking whether it worked in flat line ends.  For a pen size of 1, flat line ends shouldn't matter at all.  For a large pen size, MOVE 0 STEPS should draw a one-pixel-wide line perpendicular to the direction the sprite is facing.

The underlying issue, I think, is whether MOVE _n_ STEPS should include both endpoints of the line, and I belive the right answer is "yes, unless in penreverse mode" (which we don't have yet, so just "yes").
